admin/create-theme/theme-form.php

```diff
@@ -163,6 +163,29 @@ public static function create_admin_form_page() {
 									<input type="file" accept=".png"  name="screenshot" id="screenshot" class="upload"/>
 								</label>
 								<br /><br />
+								<label>
+									<?php _e( 'Image Credits:', 'create-block-theme' ); ?><br />
+									<small><?php _e( 'List the credits for each image you have included in the theme. Include the image name, license type, and source URL.', 'create-block-theme' ); ?></small><br />
+									<small>
+										<?php
+										printf(
+											/* Translators: Bundled resources licenses link. */
+											esc_html__( 'All bundled resources must have GPL-compatible licenses (%s).', 'create-block-theme' ),
+											'<a href="' . esc_url( __( 'https://make.wordpress.org/themes/handbook/review/resources/#licenses-bundled-resources', 'create-block-theme' ) ) . '">read more</a>'
```

Comment:
add attribute `target="_blank"`

Reply:
Good idea! I've added this in https://github.com/WordPress/create-block-theme/pull/380/commits/34df4329e2bf719faf2185d7122271dd82b90298, I also added it to the 'read more' link in the theme tags section.